### DISCUSSION TITLE
Does this feature exist.

### DISCUSSION THREAD

**Discussion by mieyeaa** [anyproto/contributors#42] [Q&A]
A subpage can be added to a page, and then subpages can be added to the subpage, and so on.

**Comment by mieyeaa** [comment]
A page like notion has Windows for other pages.

**Comment by fuksman** [comment]
Please read [contributing guidelines](https://github.com/anyproto/contributors/blob/baa61db41701e55d1cb2a4106a307edf61e7d2c1/docs/CONTRIBUTING.md); consider asking product questions in product community.
